### DISCUSSION TITLE
I found this GP alternative and am excited to try it, but where do you host it?

### DISCUSSION THREAD

**Discussion by Billium28** [immich-app/immich#7517] [General]
Kind of surprised with the lack of explanation for how and what this is. Im pretty techy but from the homepage it seems as if its a Goog;e Photos clone alternative. But what does it run on? Unraid or is this just an android app? And how do I host the photos?

**Comment by alextran1502** [comment]
Hello, this is a self hosted solution. So you will need a machine/server to run it. You van run it from Unraid or from any Linux based distro using docker/docker-compose

**Reply by Billium28** [reply]
Thanks! If I run as docker on Unraid I would just point it to my photo albums and then through setup I would then see those albums in Immich?

**Reply by alextran1502** [reply]
We have an external library feature that you can point to an existing folder, it doesn't create albums from the folder yet 

https://immich.app/docs/features/libraries

**Reply by Billium28** [reply]
Ok thanks, this is what I missed on the docs page. Cheers
